src/types.ts

```diff
@@ -110,7 +101,7 @@ export type DeeptableCreateParams = {
 
   // A manifest listing all the tiles in the deeptable.
   // Currently this must be passed as a recursive structure.
```

Comment:
update the comment here

Reply:
Updated comment. We don't have a practice anywhere of directly expressing Arrow table schemas as types, so I'm going to skip that.